### PR TITLE
Create PostgreSQL datastore in AML

### PR DESCRIPTION
**Pull request by achalachal2003**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by achalachal2003** [comment]
@bmoore-msft for review, thanks!

File: 101-datastore-create-psql/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+		"port": {
+			"type": "string",
+			"defaultValue": "5432",
+			"metadata": {
+				"description": "Optional : The port number. Defaults to 5432"
+			}
+		},
+		"endpoint": {
+			"type": "string",
+			"defaultValue": "postgres.database.azure.com",
```
Comment (bmoore-msft) [inline 101-datastore-create-psql/azuredeploy.json:50]: see comments in other PRs and make the same changes here

Comment (bmoore-msft) [reply 101-datastore-create-psql/azuredeploy.json:50]: looks like the changes have not been pushed here...

Comment (achalachal2003) [reply 101-datastore-create-psql/azuredeploy.json]: Pushed, sorry I missed to push from local.

**Pull request merged**